Support for multiple resolutions

rlogiacco opened the pull request:
Renamed a few files and added some others: this enables support for QHD (2560 x 1440) and FHD (1960 x 1080)

isaac-jordan commented:
LGTM - any chance you could update the README to tell the user to change this setting to match their resolution (and that there is only 2 valid settings currently).

rlogiacco commented:
I've updated the README dropping two settings (`image_directory` and `mask_file`) in favour of the sole `screen_format`, along the _convention over configuration_ motto.

I've also updated the README with regards to the new property: btw, it is verified at startup to avoid confusion.

rlogiacco commented:
BTW, there's a tiny issue in your original "map key image" (now `map.jpg`): you captured also the mouse pointer, which is probably making recognition a bit harder 😉

isaac-jordan commented on obs_screen_recognition_settings.json:
```diff
@@ -1,7 +1,6 @@
     "monitor_to_capture": 1,
-    "image_directory": "./hll_map_open_detection/1440p",
-    "mask_file": "./hll_map_open_detection/mask-1440p.jpg",
+    "screen_format": "1080p",
```
1 guess for which resolution we both use ;)

isaac-jordan commented on obs_screen_recognition_script.py:
```diff
@@ -88,9 +88,13 @@ def main():
     print("Running with settings:", application_settings)
-    image_directory = application_settings["image_directory"]
-    mask_file = application_settings["mask_file"]
+    image_directory = "./hll_map_open_detection/" + application_settings["screen_format"]
+    mask_file = "./hll_map_open_detection/mask-" + application_settings["screen_format"] + ".png"
```
Hardcoding this folder makes it difficult to use this repository for a different game (which I have had 1 or 2 people ask about). Should the folder name be a setting?

rlogiacco replied:
Ok, I got an idea... have a look at my additional commits: if we also rename the script file to something like `obs_obfuscator` you can run the thing with:

```
python obs_obfuscator ./hll
```

BTW, you no longer need to be in the script folder to launch it

rlogiacco commented:
There's a lot more that can be easily added via the library (https://click.palletsprojects.com/en/7.x/) like converting the `show_debug_window` setting into a launch flag (so that you don't have to touch any file to change the behaviour)

```
$ python obs_obfuscator.py --help
Usage: obs_obfuscator.py [OPTIONS] RESOURCE_DIR

Options:
  --show-debug-window
  --help               Show this message and exit.

```

So that you end up with:
```
$ python obs_obfuscator.py .\hll
```

rlogiacco commented:
looks like you need some time to decide if you wish to accept my PR, so I'm creating a dedicated branch and a new PR #8

isaac-jordan commented:
> looks like you need some time to decide if you wish to accept my PR, so I'm creating a dedicated branch and a new PR #8

Sorry, just didn't find the time to review! Appreciate your contributions though!
